### PR TITLE
Adding interactive marker proxy to launchfile.

### LINKED ISSUE #1: Add interactive marker proxy as an optional node
This is required to visualize interactive markers; see https://github.com/jstnhuang/ros-rviz/wiki/User-guide.

### PR DESCRIPTION
**Pull request by jubeira**
Closes #1.

@chapulina PTAL.

image

### CODE REVIEW

File: launch/rvizweb.launch [hunk 1]
```diff
@@ -32,4 +39,11 @@
     <node name="tf2_web_republisher" pkg="tf2_web_republisher" type="tf2_web_republisher" respawn="false" output="screen"/>
   </group>
 
+  <group if="$(arg interactive_markers)">
+    <node pkg="interactive_marker_proxy" type="proxy" name="interactive_marker_proxy">
```
Comment (chapulina) [inline launch/rvizweb.launch:43]: We should add `interactive_marker_proxy` as a dependency on `package.xml`

**Review by chapulina — CHANGES_REQUESTED** [review]
Thanks, works for me! Mind also adding a section to the readme explaining how to run a basic example? Similar to the URDF example, you could install and run `rosrun interactive_marker_tutorials basic_controls`.

**Comment by jubeira** [comment]
Thanks @chapulina; I've just sent an update, PTAL.

**Review by chapulina — APPROVED** [review]
Looks good, thanks!

**Pull request merged**